[2023-05-26 05:54:25,330] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tercera_entrega_ecentoz.db_connect scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2023-05-26 05:54:25,366] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: tercera_entrega_ecentoz.db_connect scheduled__2023-01-02T00:00:00+00:00 [queued]>
[2023-05-26 05:54:25,368] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-05-26 05:54:25,374] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-05-26 05:54:25,375] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-05-26 05:54:25,412] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): db_connect> on 2023-01-02 00:00:00+00:00
[2023-05-26 05:54:25,425] {standard_task_runner.py:52} INFO - Started process 207 to run task
[2023-05-26 05:54:25,577] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'tercera_entrega_ecentoz', 'db_connect', 'scheduled__2023-01-02T00:00:00+00:00', '--job-id', '2', '--raw', '--subdir', 'DAGS_FOLDER/dag_tercera_entrega.py', '--cfg-path', '/tmp/tmprda02qxa', '--error-file', '/tmp/tmpmfvq3c44']
[2023-05-26 05:54:25,601] {standard_task_runner.py:80} INFO - Job 2: Subtask db_connect
[2023-05-26 05:54:26,245] {task_command.py:371} INFO - Running <TaskInstance: tercera_entrega_ecentoz.db_connect scheduled__2023-01-02T00:00:00+00:00 [running]> on host f4aa0d8de4f3
[2023-05-26 05:54:27,023] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ecentoz
AIRFLOW_CTX_DAG_ID=tercera_entrega_ecentoz
AIRFLOW_CTX_TASK_ID=db_connect
AIRFLOW_CTX_EXECUTION_DATE=2023-01-02T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-02T00:00:00+00:00
[2023-05-26 05:55:20,422] {python.py:173} INFO - Done. Returned value was: None
[2023-05-26 05:55:20,484] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=tercera_entrega_ecentoz, task_id=db_connect, execution_date=20230102T000000, start_date=20230526T055425, end_date=20230526T055520
[2023-05-26 05:55:20,553] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-26 05:55:20,594] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
